walk-in-interview/src/main/webapp/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="style.css">
+  <script src="script.js"></script>
+</head>
+
+<body>
+  <div class="header-container">
+    <button onclick="location.href='/walk-in-interview/src/main/webapp/new-post.html'">New Post</button>
+    <div>Find Walk-in Interviews</div>
+    <button>Create an Account</button>
+  </div>
+  <div class="content-container">
+    <div id="homepage-map">map here</div>
+    <form class="filters-container">
+      <div>
+        Sort by:
+        <select id="sort-by-select" name="sort-by-select">
+          <option disabled value="">Select</option>
+          <option selected value="dist-asc">Distance (low to high)</option>
```

Comment:
Can u take a screen shot of this UI here? Seems that it will make "Distance (low to high)" default and "Select" one of the useless option; but i suppose what you want to do is show "Select" as default but cannot be chosen?

Reply:
I think I'll remove the "select" option and just leave the default as "distance (low to high)" since that is what we specified on the design doc.

Reply:
Sure